fossunited/www/fosshack/2026/index.py

```
@@ -0,0 +1,180 @@
+        group_by="reference_name",
+    )
+
+    # Convert to dicts
+    attending_dict = {item["localhost"]: item["count"] for item in attending_counts}
+    pending_dict = {item["localhost"]: item["count"] for item in pending_counts}
+    likes_dict = {item["localhost"]: item["count"] for item in likes_counts}
+
+    # Process LocalHosts and group by city
+    cities_dict = {}
+    for host in localhosts:
+        host["route"] = f"/{l.route}"
```

Ruff (0.14.8)

Undefined name `l`

(F821)